src/drive/web/modules/upload/index.js

```diff
@@ -36,8 +41,11 @@ const status = action => {
   }
 }
 
-const item = (state, action) =>
-  Object.assign({}, state, { status: status(action) })
+const item = (state, action = { isUpdate: false }) => ({
+  ...state,
+  isUpdate: action.isUpdate,
```

Comment:
isUpdate = the action dispatched when we need to update the doc? Or is it a boolean to know if the item has been uploaded after a conflict ? 

Can we have a comment ?

Reply:
:+1:

Reply:
It's a boolean to know if it is an update after a conflict. At first, I called it `isConflict` but it's not good. Maybe `isOverwrite` but I still feel that `isUpdate` is better since we use "update" in the messages we display to the user. We could use `action.status` for this and modify the `status()` function above to return a different status if the file has been updated. Something like : 

```
    case RECEIVE_UPLOAD_SUCCESS:
      return action.status === 'update' ? UPDATED : LOADED
```

Thus, we could return `getLoaded` and use `filterByStatus()` in `getUploaded` / `getUpdated` selectors. I will need to change `getSuccessful` too if we do this.

Reply:
If we can do with the current attributes, I would prefer this (suggested) approach